sentinelsat/sentinel.py

```diff
@@ -939,24 +885,51 @@ def check_bounds(geometry):
     wkt = re.sub(r'(?<!\d) ', '', wkt)
     return wkt
 
-
+def placename_to_wkt(placename):
+    """Maps the placename to rectangular bounding extents using Nominatim API and converts to Well-Known-Text.
+    Parameters
+    ----------
+    placename : str
+	the query to geocode
+        
+    Returns
+    -------
+    polygon coordinates
+        string of comma separated coordinate tuples (lon, lat) to be used by SentinelAPI
+    """
+    rqst = requests.post('https://nominatim.openstreetmap.org/search', \
+                         params = { 'q':placename,'format':'geojson'})
+    #Check that the response from Openstreetmapserver has status code 2xx 
+    #and that the response is valid JSON.
+    _check_scihub_response(rqst)
+    try:
+        jsonlist=rqst.json()
```

Comment:
Please use consistent spaces inbetween operators, i.e.: jsonlist = rqst.json() and further below bbox = jsonlist, footprint = "POLYGON... etc.